tests/transactional/enable_selinux.pm

```diff
@@ -49,18 +72,28 @@ sub run {
         }
     }
 
-    assert_script_run('selinuxenabled');
-    validate_script_output("getenforce", sub { m/Enforcing/ });
-    validate_script_output("sestatus", sub { m/Current mode:.*enforcing/ });
-    validate_script_output("sestatus", sub { m/Mode from config file:.*enforcing/ });
-    record_info('SELinux', script_output('sestatus'));
-    record_info('Audit report', script_output('aureport'));
-    record_info('Audit denials', script_output('aureport -a', proceed_on_failure => 1));
-    upload_logs($audit_log);
+    check_enforcing;
+
+    # disable and re-enable SELinux
+    assert_script_run "sed -i -e 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config";
```

Comment:
FYI there are few tests which modify `SELINUX` variable in `/etc/selinux/config`, including `sub set_sestatus` in lib/selinuxtest.pm. It'd be nice do do cleanup one day.